apps/sms/views/new_message/style/recipients.css

```diff
@@ -97,7 +97,7 @@ Do not remove.
   width: auto;
 }
 
-.recipient[contenteditable].email {
+.recipient[contenteditable=false].email {
```

Comment:
The difference is that when you tap on recipients it gains ```.bb-editable [contenteditable="true"]:hover``` style that applies "better" padding, with backspace we're not hovering pill. I personally don't really like like that "hover" changes padding, what do you think?